webapp/src/DOMJudgeBundle/Controller/Jury/BalloonController.php

```diff
@@ -165,17 +163,16 @@ public function indexAction(Request $request, Packages $assetPackage, KernelInte
                 $comments[] = 'first for problem';
             }
 
-            $balloondata['awards']['value'] = implode('; ', $comments);
+            $balloondata['awards'] = implode('; ', $comments);
 
             if ( $balloon->getDone() ) {
                 $cssclass = 'disabled';
                 $balloonactions = [[]];
-                $balloondata['status']['value'] = '<i class="far fa-check-circle"></i>';
-                $balloondata['status']['sortvalue'] = '1';
+                $balloondata['done'] = True;
```

Comment:
Nitpick: I think our current coding standard says these should be `true`/`false` without a capital first letter.